settings.example.ini

```diff
@@ -0,0 +1,19 @@
+# This is an example configuration file for Gajana.
+# Copy this file to 'settings.ini' and fill in your actual values,
+# or run `python src/main.py --setup` to generate it interactively.
+
+[gcp]
+service_account_key_file = secrets/google.json
+sheets_id = YOUR_GOOGLE_SHEET_ID
+drive_folder_id = YOUR_GOOGLE_DRIVE_FOLDER_ID
+
+[database]
+db_file_path = data/gajana.db
+
+[data]
+parser_configs_dir = data/configs
```

Comment:
Do we want to make these configurable or leave them truly as internal constants, and guide the user to place the files always there?